src/LeanCode.ContractsGenerator/Analyzers/InternalStructureCheck.cs

```diff
@@ -14,15 +14,15 @@ public override IEnumerable<AnalyzeError> AnalyzeTypeRef(AnalyzerContext context
         }
     }
 
-    public override IEnumerable<AnalyzeError> AnalyzeValueRef(AnalyzerContext context, ValueRef vr)
+    public override IEnumerable<AnalyzeError> AnalyzeValueRef(AnalyzerContext context, ValueRef valueRef)
```

Comment:
Yhh...

Reply:
Same as above.

Well, this one I renamed manually to be more consistent with other methods.